BlueShift-iOS-SDK/InApps/BlueShiftNotificationModalViewController.m

```diff
@@ -486,7 +487,12 @@ - (CGRect)positionNotificationView {
     frame.size = size;
     notificationView.autoresizingMask = UIViewAutoresizingNone;
     
-    CGSize screenSize = self.window.bounds.size;
+    CGSize screenSize = CGSizeZero;
+    if (self.window) {
+        screenSize = self.window.bounds.size;
+    } else {
+        screenSize = [[UIScreen mainScreen] bounds].size;
+    }
```

Comment:
can we move this code to a method and use it above as well? to get width and height?